src/main/java/seedu/address/model/item/SupplyItem.java

```diff
@@ -16,6 +17,10 @@
  */
 public class SupplyItem {
 
+
+    public static final String VALIDATION_REGEX = "(?![0-9]+$)[^\\s].*";
```

Comment:
For this check, you might want to define it in the `Developer Guide` so that others can have more clarity on this